ImmuDB4Net/ImmuClient.cs

```diff
@@ -1301,11 +1820,27 @@ public async Task CompactIndex()
     // ========== HISTORY ==========
     //
 
+    /// <summary>
+    /// Retrieves the value history of a specific key
```

Comment:
```suggestion
    /// Retrieves the history of changes of a specific key
```
The history can also contain deletions etc.

Reply:
ok